tests/conftest.py

```diff
@@ -0,0 +1,45 @@
+"""pytest plugin configuration.
+
+https://docs.pytest.org/en/latest/writing_plugins.html#conftest-py-plugins
+"""
+# Third-Party Libraries
+import pytest
+
+MAIN_SERVICE_NAME = "example"
```

Comment:
I know we're not testing (yet), but for consistency with the `docker-compose.yml`.

```suggestion
MAIN_SERVICE_NAME = "pshtt_reporter"
```